cmd/crc/cmd/status.go

```diff
@@ -90,17 +83,31 @@ func runStatus(writer io.Writer, client client, cacheDir, outputFormat string) e
 		encoder.SetIndent("", "  ")
 		return encoder.Encode(status)
 	}
-	return printStatus(writer, status, statusFormat)
+	return printStatus(writer, status)
 }
 
-func printStatus(writer io.Writer, status interface{}, statusFormat string) error {
-	tmpl, err := template.New("status").Parse(statusFormat)
-	if err != nil {
-		return fmt.Errorf("Error creating status template: %s", err.Error())
+func printStatus(writer io.Writer, status Status) error {
+	w := tabwriter.NewWriter(writer, 0, 0, 1, ' ', 0)
+	lines := []struct {
+		left, right string
+	}{
+		{"CRC VM", status.CrcStatus},
+		{"OpenShift", status.OpenShiftStatus},
+		{"Disk Usage", status.DiskUsage},
+		{"Cache Usage", status.CacheUsage},
+		{"Cache Directory", status.CacheDir},
```

Comment:
I would also like some more details about this change, like what is the benefit of using tabwritter over templates.

Reply:
In the following commit, I change the struct to remove strings and have raw numbers for usage. It was easier to build the original template by hand than with template.

Reply:
@guillaumerose Can you put that in the commit log the benefit it is going to provide moving forward or right now.

Reply:
I added the why in the commit log